src/linter-jscs.js

```diff
@@ -105,7 +108,11 @@ export default class LinterJSCS {
             this.fixString(editor);
           }
         }
-      });
+      }));
+      currentEditor.add(editor.onDidDestroy(() => {
+        currentEditor.dispose();
```

Comment:
Should this be `editor.dispose()`? Or maybe `this.textEditors.get(editor.id).dispose()`?